RadarSDK/Radar.h

```diff
@@ -431,17 +439,47 @@ typedef void(^ _Nonnull RadarRouteCompletionHandler)(RadarStatus status, RadarRo
 
  @param near The location to search.
  @param radius The radius to search, in meters. A number between 100 and 10000.
- @param tags An array of tags to filter. See https://radar.io/documentation/geofences
+ @param tags An array of tags to filter.
  @param limit The max number of geofences to return. A number between 1 and 100.
  @param completionHandler A completion handler.
 */
 + (void)searchGeofencesNear:(CLLocation *)near
                      radius:(int)radius
-                       tags:(NSArray * _Nullable)tags
+                       tags:(NSArray *_Nullable)tags
                       limit:(int)limit
           completionHandler:(RadarSearchGeofencesCompletionHandler)completionHandler
     NS_SWIFT_NAME(searchGeofences(near:radius:tags:limit:completionHandler:));
 
+/**
+ Gets the device's current location, then searches for points near that location, sorted by distance.
+
+ @param radius The radius to search, in meters. A number between 100 and 10000.
+ @param tags An array of tags to filter.
+ @param limit The max number of points to return. A number between 1 and 100.
+ @param completionHandler A completion handler.
+*/
++ (void)searchPointsWithRadius:(int)radius
+                          tags:(NSArray *_Nullable)tags
```

Comment:
I know this isn't the case on the other search endpoints, but any reason why we don't type this `NSArray`? we know it's gonna be `NSArray<NSString *>`, right?